xhr.bs

```diff
@@ -1039,7 +1039,7 @@ method must run these steps:
  <a for=body>total bytes</a>.
 
  <li><p>If the <a>synchronous flag</a> is unset, <a>fire a progress event</a> named
- <a event><code>progress</code></a> with <var>transmitted</var> and <var>length</var>.
+ <a event><code>progress</code></a> at [=this=] with <var>transmitted</var> and <var>length</var>.
```

Comment:
The intended target isn't clear in context. Firefox, Chrome, Edge, and Safari all fire this event at the request object.